source/application/CLS/CLSAppController.h

```diff
@@ -21,10 +21,55 @@ class CLSAppController : public AMAppController
 		BioXASSideBeamlineId = 7,
 		BioXASMainBeamlineId = 8,
 		BioXASImagingBeamlineId = 9,
-		SXRMBBeamlineId = 10
+		SXRMBBeamlineId = 10,
 		// new beamlines need to be added here ...
+		CLSMAXBeamlineId
 	};
 
+	/// returns the beamline name of a given beamline Id
+	static QString beamlineName(CLSAppController::CLSBeamlineID beamline)
+	{
+		QString name;
+		switch (beamline) {
+		case BlankFacility:
+			name = "";
+			break;
+		case PreserverdALS801Beamline:
```

Comment:
I'm not super familiar with how the beamline IDs are used. Is there a reason why a non-CLS beamline is being considered in CLSAppController?